content/docs/2.3/scalers/cassandra.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Cassandra"
+availability = "v2.3+"
+maintainer = "Community"
+description = "Scale applications based on Cassandra query results."
+layout = "scaler"
+go_file = "cassandra_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `cassandra` trigger that scales based on the results of a Cassandra query result.
+
+```yaml
+triggers:
+  - type: cassandra
+    metadata:
+      username: "cassandra"
+      clusterIPAddress: "cassandra.default.svc.cluster.local:9042"
+      consistency: "Quorum" # Optional: If not set by the user the default value will be `gocql.One`.
+      protoVersion: "4" # Optional: If not set by the user the default value will be `4`.
+      keyspace: "testing" # Optional: Used for generating the metricName.
+      query: "SELECT COUNT(*) FROM testing.test_table;"
+      targetQueryValue: "1"
+      metricName: "test_table-count" # Optional: If not set by the user the generated value would be `cassandra-<KEYSPACE>`, or if keyspace is not set either the default value would be just `cassandra`.
+    authenticationRef:
+      name: keda-trigger-auth-cassandra-secret
+```
+
+**Parameter list:**
+
+- `username` - The username credential for connecting to the Cassandra instance.
+- `clusterIPAddress` - The IP address or the host name of the Cassandra instance, with port number (optional).
```

Comment:
@zroubalik We had a scaler where we support full name and name + port separately, do you recall where we changed this? We might want to align here.

Reply:
I believe that it might be MSSQL scaler since the port is indicated as optional in the doc here:  https://github.com/kedacore/keda-docs/blob/master/content/docs/2.3/scalers/mssql.md
So you mean that we should also support `port` as optional, right?

Reply:
> I believe that it might be MSSQL scaler since the port is indicated as optional in the doc here: https://github.com/kedacore/keda-docs/blob/master/content/docs/2.3/scalers/mssql.md
> So you mean that we should also support `port` as optional, right?

Is this right? @tomkerkhove

Reply:
I'd love to align this indeed, what do you think @zroubalik ?

Reply:
Yeah, I'd prefer to have the ability to specify the port in an optional field.